### ISSUE TITLE
Issue when applying the plugin

### ISSUE THREAD

**Issue by odvratnozgodan** [hyperdevs-team/poeditor-android-gradle-plugin#72]
**SW details (please complete the following information):**
 - IDE version: Android Studio Giraffe | 2022.3.1 Patch 2
 - Plug-in Version 4.1.0

**Summary and background of the bug**
When applying the plugin to a project I get the following error:

```
A problem occurred configuring project ':app'.
> Cannot query the value of extension 'poEditor' property 'enabled' because it has no value available.
```

**Steps to reproduce**
1. Sync project with gradle files

I have replicated this on my work project and also tired it in a new bare bones Android project. I have attached the project for replication purposes.

[PlaygroundApplication.zip](https://github.com/hyperdevs-team/poeditor-android-gradle-plugin/files/13524313/PlaygroundApplication.zip)

If you need any more info let me know.

**Expected behavior**
The sync should pass without error.

**Additional context**

**Comment by viroth-ty** [comment]
It work for me 
```
plugins {
    id("com.android.application") version "8.1.2" apply false
    id("org.jetbrains.kotlin.android") version "1.9.10" apply false
    id("com.hyperdevs.poeditor") version "4.0.0" apply false
}
```

**Comment by odvratnozgodan** [comment]
Have you tried to replicate it in the project that I attached? It contains the same code as on the README.md document of the repo and it's a basic Android Studio project(no extra code or libraries).
I was trying to make the plugin work in my project but then decided to create a clean project and try to set it up there. That's when I found that something else was wrong with the plugin(or the documentation inside the readme).

**Comment by adriangl** [comment]
Hi @odvratnozgodan!

Thanks for the bug report and the sample project, I've been able to pinpoint two issues:
* The first issue is that, thanks to the sample project, I found a bug in 4.1.0 that made the `enabled` property to not be set properly if you don't set it up explicitly. This will be fixed in the next release.
* The second issue is that  the plugin requires Gradle 8.2 or above (not the AGP, but Gradle). I'll make this clear in the README file.

Also, just a note (it may be because it's just a sample project, but just in case): you should fix the `settings.gradle.kts` in the sample project so it properly picks the plug-in version that you define in the root `build.gradle.kts`. If you force the version in `settings.gradle.kts` the version set in `build.gradle.kts` won't be honored. 

```kotlin
pluginManagement {
    resolutionStrategy {
        eachPlugin {
            if (requested.id.id == "com.hyperdevs.poeditor") {
                useModule("com.github.hyperdevs-team:poeditor-android-gradle-plugin:${requested.version}")
            }
        }
    }
}
```

**Comment by adriangl** [comment]
Hi @odvratnozgodan!
The issue should be fixed in version `4.1.1`, please check if it's fixed!
